### ISSUE TITLE
JSON messages are being serialized twice when using Schema Registry

### ISSUE THREAD

**Issue by SmithJohnTaylor** [ShadowTraffic/shadowtraffic-examples#3]
Producing to Confluent Cloud w/ Schema Registry with these configs:
```
"connections": {
    "dev-kafka": {
      "kind": "kafka",
      "producerConfigs": {
        "bootstrap.servers": "",
        "key.serializer": "org.apache.kafka.common.serialization.StringSerializer",
        "value.serializer": "io.confluent.kafka.serializers.json.KafkaJsonSchemaSerializer",
        "schema.registry.url": "",
        "security.protocol": "SASL_SSL",
        "sasl.mechanism": "PLAIN",
        "sasl.jaas.config": "org.apache.kafka.common.security.plain.PlainLoginModule required username='' password='';",
        "basic.auth.credentials.source": "USER_INFO",
        "basic.auth.user.info": ":"
      }
    }
  }
```
Yields this payload:
```
"{\"op_type\":\"U\",\"before\":{\"COMPANY\":6026,\"JOB_CODE\":\"1432\",\"ACTIVE_FLAG\":\"A\",\"JOB_CLASS\":\"S  \"},\"after\":{\"COMPANY\":6041,\"JOB_CODE\":\"1432\",\"JOB_CLASS\":\"RMD\",\"ACTIVE_FLAG\":\"X\"}}"
```

This causes issues when trying to read/deserialize the message:
```
"Failed to deserialize value from topic: test. Can't convert type. sourceType: TextNode, requiredType: STRUCT<BEFORE STRUCT<JOB_CODE VARCHAR, ACTIVE_FLAG VARCHAR, JOB_CLASS VARCHAR, COMPANY BIGINT>, OP_TYPE VARCHAR, AFTER STRUCT<JOB_CODE VARCHAR, ACTIVE_FLAG STRUCT<IO.CONFLUENT.CONNECT.JSON.ONEOF.FIELD.0 VARCHAR, IO.CONFLUENT.CONNECT.JSON.ONEOF.FIELD.1 VARCHAR>, JOB_CLASS VARCHAR, COMPANY BIGINT>>, path: $",
```

**Comment by MichaelDrogalis** [comment]
Thanks for reporting @SmithJohnTaylor! I'm investigating now.

**Comment by MichaelDrogalis** [comment]
Reproduced. Working on a fix now.

**Comment by MichaelDrogalis** [comment]
@SmithJohnTaylor Got a fix out—try `0.3.10`. Sorry about that!
